themes/models.py

```diff
@@ -0,0 +1,69 @@
+from django.db import models
+from users.models import User
+
+
+class Themes(models.Model):
```

Comment:
when writing models, always use the singular form for the name. it represents an Instance/Object not a collection of objects.